apps/web/src/routes/repositories/+page.svelte

```diff
@@ -1,7 +1,30 @@
 <script lang="ts">
+	import { UserService } from '$lib/user/userService';
+	import { getContext } from '@gitbutler/shared/context';
+	import { ProjectService } from '@gitbutler/shared/organizations/projectService';
+	import { getAllUserProjects } from '@gitbutler/shared/organizations/projectsPreview.svelte';
+	import { AppState } from '@gitbutler/shared/redux/store.svelte';
+
+	const appState = getContext(AppState);
+	const projectService = getContext(ProjectService);
+	const userService = getContext(UserService);
+
+	const user = $derived(userService.user);
+	const username = $derived($user?.login);
+
+	const userProjects = $derived(getAllUserProjects(username ?? 'bla', appState, projectService));
 </script>
 
 <h2>Your projects:</h2>
 
+{#each userProjects.current as project (project.id)}
+	{#if project.status === 'found'}
+		<div>
+			<p>{project.value.name}</p>
+			<p>{project.value.createdAt}</p>
+		</div>
+	{/if}
```

Comment:
We have a <Loadable> component which handles the states and passes the value back to you.